src/confluent_kafka/src/Admin.c

```diff
@@ -2835,6 +3195,89 @@ static PyObject *Admin_c_DescribeConsumerGroupsResults_to_py(
         return NULL;
 }
 
+static PyObject *Admin_c_ScramMechanism_to_py(rd_kafka_ScramMechanism_t mechanism){
+        PyObject *result = NULL, *value_dict = NULL;
+        PyObject *args = NULL;
+        value_dict = PyDict_New();
+        cfl_PyDict_SetInt(value_dict, "value",(int) mechanism);
+        args = PyTuple_New(0);
+        result = PyObject_Call(cfl_PyObject_lookup("confluent_kafka.admin",
```

Comment:
I think we need to decrease refcount for the new reference returned by `cfl_PyObject_lookup` function.